homeassistant/components/sensor/pollen.py

```diff
@@ -125,18 +127,23 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         'allergy_index_data': AllergyIndexData(client),
         'disease_average_data': DiseaseData(client)
     }
+    classes = {
+        'AllergyAverageSensor': AllergyAverageSensor,
+        'AllergyIndexSensor': AllergyIndexSensor
+    }
 
     for data in datas.values():
         data.update()
 
     sensors = []
     for condition in config[CONF_MONITORED_CONDITIONS]:
         name, sensor_class, data_key, params, icon = CONDITIONS[condition]
-        sensors.append(globals()[sensor_class](
+        sensors.append(classes[sensor_class](
             datas[data_key],
             params,
             name,
-            icon
+            icon,
+            adler32(str(config[CONF_ZIP_CODE]).encode('utf-8'))
```

Comment:
Why would you do this? Why not just pass the zip ?

Reply:
Fighting a cold and my brain was locked on what I did for AirVisual. Will simplify.